contracts/test/regIndex-test.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable camelcase */
+
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { ethers } from "hardhat";
+import { encodeRegIndex } from "../shared/helpers/bundleCompression";
+import { RegIndex, RegIndex__factory } from "../typechain-types";
```

Comment:
I've been doing `as` renames on Typechain factory imports to remove the need for eslint disables. Keeps camelcase naming consistent throughout rest of code.

```suggestion
import { expect } from "chai";
import { BigNumber } from "ethers";
import { ethers } from "hardhat";
import { encodeRegIndex } from "../shared/helpers/bundleCompression";
import { RegIndex, RegIndex__factory as regIndexFactory } from "../typechain-types";
```

Reply:
https://github.com/web3well/bls-wallet/pull/556